console

```diff
@@ -16,6 +20,29 @@ $application = require __DIR__ . '/res/bootstrap.php';
 
 $cli = $application->cli();
 
+$cli->command('serve', function (\Stratify\Http\Application $http, ContainerInterface $container, OutputInterface $output) {
+    $loop = React\EventLoop\Factory::create();
+
+    $db = $container->get(Connection::class);
+
+    $server = new Server(function (ServerRequestInterface $request) use ($http, $db) : ResponseInterface {
+        // Reset the DB connection
+        $db->close();
```

Comment:
I found this analysis of the issue:

http://brady.lucidgene.com/articles/pdo-lost-mysql-connection-error

There are some ideas for other approaches in the comment section...